core_impl/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -288,7 +301,7 @@ private void testRecordInterval(
         viewManager.getView(VIEW_NAME).getAggregationMap(),
         ImmutableMap.of(
             Arrays.asList(VALUE),
-            StatsTestUtil.createAggregationData(MEAN, initialValues)),
+            StatsTestUtil.createAggregationData(MEAN, MEASURE_LONG, initialValues)),
```

Comment:
Should this be `measure` instead of `MEASURE_LONG`, since this method is called with both subclasses?

Reply:
Good catch, fixed. Also added tests for recording sum stats for interval view.